packages/web-pkg/src/composables/piniaStores/avatars.ts

```diff
@@ -0,0 +1,52 @@
+import { defineStore, storeToRefs } from 'pinia'
+import { ref, unref } from 'vue'
+import { useUserStore } from './user'
+
+export type AvatarMap = Record<string, string>
+
+export const useAvatarsStore = defineStore('avatars', () => {
+  const avatarMap = ref<AvatarMap>({})
```

Comment:
Any reason against inlining the type? You don't use the exported `AvatarMap` anywhere outside of this file...

```suggestion
  const avatarMap = ref<Record<string, string>>({})
```